client/stylesheets/puzzle.scss

```diff
@@ -350,9 +351,21 @@ table.puzzle-list {
   display: inline-block;
 }
 
+.puzzle-metadata-action-row {
+  flex-wrap: wrap;
+}
+
 .puzzle-metadata .puzzle-metadata-external-link-button, .puzzle-metadata .gdrive-button {
   display: inline-block;
   font-weight: bold;
+  white-space: nowrap;
+  /* On very small touch screens or split view, desperately try to prevent line wrap */
+  /* Revisit this */
+  @media (max-width: 320px) and (pointer: coarse) {
+    svg {
```

Comment:
Is this just hiding the font icon?

Reply:
Yup.  It's neither the logic nor the behavior I'd ideally want, but it's something I could patch together in a few minutes this morning and it does solve the most common (by far) case.